### PR TITLE
add custom isclose function

### PR DESCRIPTION
**Pull request by vanyakosmos**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def readme():
         'Topic :: Utilities',
     ],
     keywords=['doctest', 'rst', 'pytest', 'py.test'],
-    install_requires=['six', 'pytest>=3.0', 'numpy>=1.10'],
+    install_requires=['six', 'pytest>=2.8.0'],
```
Comment (bsipocz) [inline setup.py]: keep  the pytest requirement

File: pytest_doctestplus/output_checker.py [hunk 2]
```diff
@@ -189,3 +187,9 @@ def output_difference(self, want, got, flags):
         # new-style class.
         return self._original_output_checker.output_difference(
             self, want, got, flags)
+
+
+def isclose(ng, nw, rtol=1e-05, atol=1e-08):
```
Comment (bsipocz) [inline pytest_doctestplus/output_checker.py]: could we still rely on numpy when it's available, most packages that use this plugin has it anyway (thus the reason this wasn't very high on our todo)?

it's not a strong preference, I leave it with @astrofrog to decide

File: setup.py (hunk 1 above)
Comment (bsipocz) [reply setup.py]: (a rebase would resolve that, I see this was coming from commit cherry-picked from the other PR)

**Comment by bsipocz** [comment]
The travis matrix needs some updates, would you mind allowing pushes from maintainers? It should be a tick box on the right hand side of the PR.

**Comment by vanyakosmos** [comment]
> Would you mind allowing pushes from maintainers? It should be a tick box on the right hand side of the PR.
It is already checked. 

Btw, how can I run tests locally? pytest always throw `pytest: error: unrecognized arguments: --doctest-plus` on every test. I tried to install package locally (`pip install -e .`) and from pypi, but no luck.

**Comment by bsipocz** [comment]
`pytest --doctest-plus` should work 

(I did a `pip install -e .` before that, the travis config still has `python ./setup.py install`, both should work really)

**Comment by bsipocz** [comment]
Weird, I still get the permission denied error. Anyway, I've pushed a copy of this branch with my extra commit on top  here:

https://github.com/bsipocz/pytest-doctestplus/tree/vanyakosmos_allclose

you  could cherry-pick, or copy  the top commit that should be a better test.

**Comment by vanyakosmos** [comment]
> ERROR: No matching distribution found for pytest==3.9

3.9.3 is available tho. Should I edit version in .travis.yml? Or just remove matrix entry with that pytest version?
Kinda strange that [here](https://travis-ci.org/astropy/pytest-doctestplus/jobs/586337000) is the same setup but pytest was installed successfully.

**Comment by bsipocz** [comment]
No worries, it's an upstream issue that I'm looking into atm.

**Comment by bsipocz** [comment]
Thank you @vanyakosmos!

**Pull request merged**